package.json

```diff
@@ -19,6 +20,7 @@
     "deep-freeze": "0.0.1",
     "eslint-config-airbnb-base": "^12.1.0",
     "lodash": "^4.17.5",
+    "moment": "^2.22.2",
```

Comment:
I added another comment elsewhere, but can we us date-fns (which we are already including) instead of moment? (Or switch from date-fns to moment)?